src/utils/require-template.js

```diff
@@ -19,30 +19,34 @@ function isValidTemplate(template){
   ].every((method) => template[method]);
 }
 
+const getOcTemplate = (path) => {
+  if (require.cache && !!require.cache[path]) {
+    delete require.cache[path];
+  }
+  return require(path);
+};
 
-module.exports = function(template) {
+module.exports = function (template) {
   let ocTemplate;
   const localTemplate = path.join(__dirname, '../../', 'node_modules', template);
   const relativeTemplate = path.resolve('.', 'node_modules', template);
 
   try {
-    if (require.cache && !!require.cache[localTemplate]) {
-      delete require.cache[localTemplate];
-    }
-    ocTemplate = require(localTemplate);
-  } catch(err) {
+    ocTemplate = getOcTemplate(template);
```

Comment:
I've just tried it, and I can't publish due to this: https://github.com/npm/npmlog/issues/48

![imatge](https://cloud.githubusercontent.com/assets/153305/26408824/20ce750a-409f-11e7-9fa1-43e72a0b3318.png)

😕